### ISSUE TITLE
Media and other keys autoscroll the terminal

### ISSUE THREAD

**Issue by tgross35** [microsoft/terminal#5366]
# Description
Keys that aren't meant to insert characters seem to jump the terminal to the bottom where input is being awaited. This includes media buttons.

# Steps to reproduce
Open any terminal tab (tested with both Ubuntu and Powershell) and run anything to fill up the screen (e.g. run `ls` 5 times). Scroll up until the text input area is off the page. 

Press any non-typing character that isn't a modifier (tested with "calculator" key, play/pause, and volume control keys) 

# Expected Behavior
These keys should have no effect on terminal

# Actual behavior
Terminal scrolls to the bottom of the page where a new line is 

![ezgif-7-c7e99acf7c07](https://user-images.githubusercontent.com/13724985/79384807-8d5e8e80-7f35-11ea-943a-c395332eff09.gif)

In this video I press the mute key or the calculator key, causing the terminal to jump

# Version Info
Windows Terminal
Version: 0.10.781.0

**Comment by DHowett-MSFT** [comment]
Yeah, that's a fair pull. We should only scroll on "input" if there actually was input.

**Comment by Decimation** [comment]
Any updates on this? This is still a problem on the latest version.

**Comment by spinstartshere** [comment]
This is also happening to me when clicking within a PS window to highlight text.

**Comment by spinstartshere** [comment]
This is also happening to me when clicking within a PowerShell window to highlight text.

**Comment by lhecker** [comment]
Pretty sure this is due to the `IsInputKey` function. It needs to be swapped to a whitelist, or we need to add the volume vkey. Tagged it up for 1.26, because it's probably easy to fix.
